src/aave-v2/SupplyVault.sol

```diff
@@ -10,12 +10,21 @@ import {SupplyVaultBase} from "./SupplyVaultBase.sol";
 /// @custom:contact security@morpho.xyz
 /// @notice ERC4626-upgradeable Tokenized Vault implementation for Morpho-Aave V2.
 contract SupplyVault is ISupplyVault, SupplyVaultBase {
-    /// UPGRADE ///
+    /// STORAGE ///
+
+    /// @dev This empty reserved space is put in place to allow future versions to add new
+    /// variables without shifting down storage in the inheritance chain.
+    /// See https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
+    uint256[50] private __gap;
```

Comment:
This is not needed for now

Reply:
why for now? Better to add then rn than forget it later no?

Reply:
Then we'd need to add this to each end-layer implementation of contracts to be coherent
OZ does not do this, it's overkill to me